apps/web-giddh/src/app/audit-logs/components/audit-logs-table/audit-logs-table.component.html

```diff
@@ -13,6 +14,13 @@ <h1>{{log.operationType}}
         <ng-container *ngIf="log?.operationType==='MOVE'">
             <span *ngIf="log && log?.details && log?.details?.entity"> - {{log?.details?.entity.name}}({{log?.details?.entity.uniqueName}})</span>
         </ng-container>
+        <!-- For account start header -->
+        <!-- For Stock start header -->
+        <ng-container *ngIf="log && log?.details && log?.details?.stockGroup && log?.details?.stockGroup?.uniqueName != undefined">
```

Comment:
Consider using shorter expressions with the help of safe navigation operator
```suggestion
        <ng-container *ngIf="log?.details?.stockGroup?.uniqueName">
```

Please use the same in below HTML code